src/routers/usersRouter.js

```diff
@@ -501,7 +503,77 @@ userRouter.get(
 userRouter.delete(
   '/notifications/:notificationId',
   checktoken,
-  deleteNotification,
+  deleteNotification
 );
 
+/**
+ * @swagger
+ *
+ * /api/users/{userID}:
+ *  delete:
+ *   summary: Deleting a user(admin and operator are allowed only)
+ *   description: This DELETE request deletes a user from the system permanently
+ *   tags:
+ *      - Users
+ *   parameters:
+ *    - in: header
+ *      name: auth
+ *      required: true
+ *      type: string
+ *      description: Enter Authorization token
+ *    - in: path
+ *      name: userID
+ *      required: true
+ *      type: string
+ *      description: Enter User ID to delete
+ *    - in: query
+ *      name: lang
+ *      schema:
+ *        type: string
+ *      description: Your preferred language
+ *   responses:
+ *    200:
+ *     description: User deleted successfully
+ *    404:
+ *     description: User not found :(
+ *    500:
+ *     description: Internal server error
+ */
+
+userRouter.delete('/:userId', checktoken, isNotDriver, deleteOne);
```

Comment:
why are we checking `isNotDriver` functionalities?

Reply:
i think the Owner should not have rights to delete his own profile